beacon-chain/core/validators/validator.go

```diff
@@ -269,7 +290,7 @@ func AddPendingValidator(
 	newValidatorRecord := &pb.ValidatorRecord{
 		Pubkey:                 pubKey,
 		RandaoCommitmentHash32: randaoCommitment,
-		Balance:                params.BeaconConfig().DepositSize * params.BeaconConfig().Gwei,
+		Balance:                params.BeaconConfig().MaxDeposit * params.BeaconConfig().Gwei,
```

Comment:
Why `MaxDeposit` here ? Shouldnt the balance be whats the deposit from the transaction log in the POW chain ?

Reply:
`AddPendingValidator` is going to get updated with the new spec soon